docs/apache-airflow/executor/kubernetes.rst

```diff
@@ -113,7 +113,7 @@ create a V1pod with a single container, and overwrite the fields as follows:
     :start-after: [START task_with_volume]
     :end-before: [END task_with_volume]
 
-Note that volume mounts, environment variables, ports, and devices will all be extended instead of overwritten.
+Note that volume mounts, environment variables, ports, and devices will all be extended instead of overwritten. Note also that the volumes and the init_containers from the spec will be extended.
```

Comment:
Good call. I'd just add these 2 to the first list though? I don't think we need it separate?

Reply:
The first note refers to the container, and the one that I have added to the spec. I will try to rephrase it a little bit.

Reply:
Please check new changes.

Reply:
That works. I'm not sure the distinction really matters though 🤷‍♂️